library/src/main/java/com/ibm/research/drl/dpt/anonymization/constraints/LDiversityMetric.java

```diff
@@ -28,9 +28,9 @@ public LDiversityMetric(List<String> sensitiveValues) {
         super(1);
         this.histograms = new ArrayList<>();
 
-        for (int i = 0; i < sensitiveValues.size(); i++) {
+        for (String sensitiveValue : sensitiveValues) {
```

Comment:
same here, should upgrade copyright to 2022?

Reply:
No real changes to this file